server/models/assessRisksAndNeeds/riskSummary.ts

```diff
@@ -1,10 +1,12 @@
 export type RiskScore = 'LOW' | 'MEDIUM' | 'HIGH' | 'VERY_HIGH'
 
 export default interface RiskSummary {
```

Comment:
Not that big a deal, but should this be a top level `Risk(s)` interface rather than `RiskSummary`, now that it's got a summary field)